airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -157,8 +157,8 @@ class ClusterGenerator:
     # pylint: disable=too-many-arguments,too-many-locals
     def __init__(
         self,
-        project_id: Optional[str] = None,
-        cluster_name: Optional[str] = None,
+        project_id: str,
+        cluster_name: str,
```

Comment:
Should a note about these arguments no longer being optional be added somewhere (UPDATING.md perhaps)?

Reply:
I added fallback for default project id. The cluster name has required any way